pkg/genericapiserver/storage_factory.go

```diff
@@ -104,12 +104,21 @@ const AllResources = "*"
 // specialDefaultResourcePrefixes are prefixes compiled into Kubernetes.
 // TODO: move out of this package, it is not generic
 var specialDefaultResourcePrefixes = map[schema.GroupResource]string{
-	schema.GroupResource{Group: "", Resource: "replicationControllers"}: "controllers",
-	schema.GroupResource{Group: "", Resource: "replicationcontrollers"}: "controllers",
-	schema.GroupResource{Group: "", Resource: "endpoints"}:              "services/endpoints",
-	schema.GroupResource{Group: "", Resource: "nodes"}:                  "minions",
-	schema.GroupResource{Group: "", Resource: "services"}:               "services/specs",
-	schema.GroupResource{Group: "extensions", Resource: "ingresses"}:    "ingress",
+	schema.GroupResource{Group: "", Resource: "replicationControllers"}:        "controllers",
+	schema.GroupResource{Group: "", Resource: "replicationcontrollers"}:        "controllers",
+	schema.GroupResource{Group: "", Resource: "endpoints"}:                     "services/endpoints",
+	schema.GroupResource{Group: "", Resource: "nodes"}:                         "minions",
+	schema.GroupResource{Group: "", Resource: "services"}:                      "services/specs",
+	schema.GroupResource{Group: "", Resource: "service"}:                       "services/specs",
```

Comment:
remove this, this was federation only and they need `schema.GroupResource{Group: "", Resource: "services"}` to map to `service`, which we can't do here